### PR TITLE
! prefix referencing templates

### PR DESCRIPTION
**Pull request by silverbackdan**
https://github.com/symfony/symfony-docs/issues/8440

### CODE REVIEW

**Comment by yceruto** [comment]
@silverbackdan thanks for worrying about this! :)

**Comment by silverbackdan** [comment]
No probs, just stumbled across the merged PR and then the docs PR when looking for the answer. If I do this in future, is 1 PR sufficient until approved in master and then PR for the other versions? Didn't really think about it before submitting them all.

File: templating/overriding.rst [hunk 1]
```diff
@@ -19,6 +19,11 @@ from the bundle to ``app/Resources/AcmeBlogBundle/views/Blog/index.html.twig``
 (the ``app/Resources/AcmeBlogBundle`` directory won't exist, so you'll need
 to create it). You're now free to customize the template.
 
+If you don't want to override the entire template but instead just want to 
+override a block, as of Symfony 3.4 you can now do with by extending the original
```
Comment (iltar) [inline templating/overriding.rst]: Usually the "as of Symfony x.x, you can" are added via the versionadded blocks. Those blocks are removed with the next major I believe.

**Comment by nicolas-grekas** [comment]
should we take over this PR? or are you ok to finish it @silverbackdan ?

File: templating/overriding.rst (hunk 1 above)
Comment (silverbackdan) [reply templating/overriding.rst]: Thanks @iltar

**Comment by silverbackdan** [comment]
@nicolas-grekas I'm sorry for leaving this - I've made the update to use the versionadd block now. I updated the wording on that file too.

**Comment by javiereguiluz** [comment]
Thanks for this contribution! Sorry we forgot about this and we didn't merge it earlier. @silverbackdan congrats on your first Symfony Docs contribution too!

**Comment by David-Crty** [comment]
Sorry but it doesn't look to be merged can't find it the in the doc :/ 
https://symfony.com/doc/current/bundles/override.html

**Comment by silverbackdan** [comment]
I think this has been improved since this was merged. It shows from version 4.1 and above but I think it was implemented in 3.4

**Comment by xabbuh** [comment]
for 3.4, this part is located in https://symfony.com/doc/3.4/templating/overriding.html

**Pull request closed without merging**